lms/djangoapps/instructor/views/api.py

```diff
@@ -2133,6 +2133,32 @@ def get_course_forums_usage(request, course_id):
             "for the status of the task. When completed, the report "
             "will be available for download in the table below."
         )
+
+        return JsonResponse({
+            "status": already_running_status
+        })
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_ora2_responses_email(request, course_id):
+    """
+    Pushes a Celery task which will aggregate ora2 responses for a course into a .csv
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
As per the discussion about opaque keys we shouldn't be using from_deprecated_string now. Here's a link to the doc:
https://github.com/edx/edx-platform/wiki/Locators%3A-Developer-Notes